x-pack/filebeat/input/netflow/input_test.go

```diff
@@ -9,11 +9,24 @@ package netflow
 import (
 	"testing"
 
-	"github.com/elastic/beats/v7/filebeat/input/inputtest"
+	"github.com/elastic/beats/v7/libbeat/tests/resources"
+
+	conf "github.com/elastic/elastic-agent-libs/config"
+	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/elastic/elastic-agent-libs/mapstr"
+
+	"github.com/stretchr/testify/require"
 )
 
 func TestNewInputDone(t *testing.T) {
-	config := mapstr.M{}
-	inputtest.AssertNotStartedInputCanBeDone(t, NewInput, &config)
+	configMap := conf.MustNewConfigFrom(mapstr.M{})
+
+	goroutines := resources.NewGoroutinesChecker()
+	defer goroutines.Check(t)
+
+	config, err := conf.NewConfigFrom(configMap)
+	require.NoError(t, err)
+
+	_, err = Plugin(logp.NewLogger("netflow_test")).Manager.Create(config)
+	require.NoError(t, err)
```

Comment:
All config/maps are empty, so you don't need the `conf.NewConfigFrom`
```suggestion
func TestNewInputDone(t *testing.T) {
	goroutines := resources.NewGoroutinesChecker()
	defer goroutines.Check(t)

	config := conf.MustNewConfigFrom(mapstr.M{})

	_, err := Plugin(logp.NewLogger("netflow_test")).Manager.Create(config)
	require.NoError(t, err)
}
```